src/test/components/__snapshots__/Root.test.js.snap

```diff
@@ -2,6 +2,115 @@
 
 exports[`app/ProfileViewWhenReady does not try to retrieve a profile when moving from from-addon to public 1`] = `<profile-viewer />`;
 
+exports[`app/ProfileViewWhenReady renders a compare home when navigating to /compare/, then loads when changing profiles 1`] = `
+<main
```

Comment:
I'm just re-iterating here that the snapshot is identical to ` src/test/components/__snapshots__/CompareHome.test.js.snap`